omod/src/main/java/org/openmrs/module/htmlformentryui/page/controller/htmlform/ViewEncounterWithHtmlFormPageController.java

```diff
@@ -26,6 +28,19 @@ public void get(@RequestParam("encounter") Encounter encounter,
 
         patient.setPatient(encounter.getPatient());
 
+        AdministrationService administrationService = Context.getAdministrationService();
+        
+        String customPrintModule = administrationService.getGlobalProperty("htmlformentryui.customPrintModule");
+        String customPrintUrl = administrationService.getGlobalProperty("htmlformentryui.customPrintUrl");
+        String customPrintTarget = administrationService.getGlobalProperty("htmlformentryui.customPrintTarget");
+        if (StringUtils.isBlank(customPrintTarget)) {
+            customPrintTarget = "_self";
+        }
+
+        model.addAttribute("customPrintModule", customPrintModule);
+        model.addAttribute("customPrintUrl", customPrintUrl);
+        model.addAttribute("customPrintTarget", customPrintTarget);
```

Comment:
See my naming convention change requests from above.

Reply:
Latest commit makes this change.